javascript/modules/client/src/context.ts

```diff
@@ -16,6 +16,7 @@ let defaultState: ContextState = {
   dao: "",
   daoFactoryAddress: "",
   web3Providers: [],
+  gasFeeReducer: 0.625,
```

Comment:
Will this value remain in the future? If s, I would suggest a clearer name, like `gasFeeEstimationFactor` or `gasFeeAverageFactor